build.gradle

```diff
@@ -136,6 +136,11 @@ test {
     }
 }
 
+compileTestJava {
```

Comment:
Why is this necessary?

Reply:
I think this was a leftover to get it working in eclipse